distutils/unixccompiler.py

```diff
@@ -261,7 +261,7 @@ def runtime_library_dir_option(self, dir):
                 # No idea how --enable-new-dtags would be passed on to
                 # ld if this system was using GNU ld.  Don't know if a
                 # system like this even exists.
-                return "-R" + dir
+                return "-Wl,-R" + dir
```

Comment:
Do the existing comments still make sense in light of this change?

Reply:
Since all three branches now supply `-Wl,-R`, probably this whole block from line 249 should be re-written to describe the new considerations.

Reply:
Good observation. Changed now.